src/Grid/RemoteDataSource.js

```diff
@@ -31,6 +31,37 @@ class RemoteDataSource {
     this._doRequest(rowsPerPage, page);
   }
 
+  getAllRecords = (count, searchText) => new Promise((resolve, reject) => {
```

Comment:
Nop, why don't reuse `doRequest`?

Reply:
Because I don't want to modify the current table items.

Reply:
Invalid reason, common. Reuse some code.

Reply:
Forgive me father for I have sinned

Reply:
😱